src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -201,6 +211,29 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
   }
 
   ngOnInit() {
+    this.columns = [
+      {
+        name: $localize`Group Name`,
+        prop: 'groupName',
+        flexGrow: 1
+      },
+      {
+        name: $localize`Status`,
+        prop: 'status',
+        flexGrow: 1,
+        cellTemplate: this.statusTmpl
```

Comment:
Rather than creating a new statusTmpl that does many stuffs, we can reuse those from the already existing implementation. 

If you want to go further and generalize the tooltipTpl as well, please move the that particular template to table.component.html where we already have a lot of templates. That way we can reuse this for other areas as well.
```suggestion

constrcutor(
      ...
      private titleCasePipe: TitleCasePipe
      )
      
       cellTransformation: CellTemplate.badge
       customTemplateConfig: {
            map: {
                 enabled: { class: 'badge-success' },
                 allowed: { class: 'badge-info' },
                 forbidden: { class: 'badge-warning' }
            }
        },
        pipe: this.titleCasePipe
        cellTemplate: this.toolTipTpl
```